src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_format.py

```diff
@@ -69,7 +75,7 @@ def transform_file_output(result):
         new_entry = OrderedDict()
         item_name = item['name']
         try:
-            _ = item['properties']['contentLength']
+            item['properties']['contentLength']  # pylint: disable=pointless-statement
```

Comment:
Files have 'contentLength' but directories don't.  Either way your change is fine.